server/views/partials/components/sentence/template.njk

```diff
@@ -8,25 +8,21 @@
 {% set sentenceDate = sentence.sentenceDate | formatDate if sentence.sentenceDate else 'N/A' %}
```

Comment:
we can get rid of this I think?? just use  sentence.sentenceDate in the offenceCard